### PR TITLE
added Japanese translation

### PR DESCRIPTION
**Pull request by j1oba**
First draft Japanese translation for review and included to the repo.

### CODE REVIEW

**Comment by mbeylin** [comment]
Hey @j1oba is this PR ready for review? If not just let me know when you're finished with it 👍

**Comment by j1oba** [comment]
> Hey @j1oba is this PR ready for review? If not just let me know when you're finished with it

@mbeylin , I reviewed again and made some corrections. This PR is ready for review.

**Comment by j1oba** [comment]
@mbeylin Another thing to mention. I saw you guys conversation in Mandarin Chinese PR, so I translated the line, accordingly.

File: src/i18n/jp_JP.json [hunk 1]
```diff
@@ -0,0 +1,968 @@
...
+      },
+      "headers": {
+        "title": "報奨のタイトル",
+        "fulfiller": "フルフィラー",
+        "date": "送信日",
+        "status": "ステータス",
+        "payment": "支払い額"
+      },
+      "zero_state": {
+        "title": "あなたの送信は {currentTab} 0 です,
```
Comment (ntkomata) [inline src/i18n/jp_JP.json]: seems missing a quote

Comment (j1oba) [reply src/i18n/jp_JP.json]: @ntkomata I fixed it. Thank you for your pointing it out.

Comment (ntkomata) [reply src/i18n/jp_JP.json]: hehe no problem

**Comment by mbeylin** [comment]
Thank you for your work! I've extended the bounty deadline; could you go to the bounty and submit a link to this PR, and I will accept it there? 

Thank you for all of your help! 👍

**Comment by j1oba** [comment]
> 
> 
> Thank you for your work! I've extended the bounty deadline; could you go to the bounty and submit a link to this PR, and I will accept it there?
> 
> Thank you for all of your help! +1

@mbeylin Thank you for your response. I will do what you mentioned.

**Pull request merged**